articles/aks/deploy-spinkube.md

```diff
@@ -0,0 +1,274 @@
+- `kube` plugin for `spin`: A `spin` CLI plugin allowing users to scaffold Kubernetes deployment manifests for Spin Apps.
+
+
+## Before you begin
+
+You must have the latest version of Azure CLI, [`kubectl`][kubectl], [Helm][helm], and [Spin CLI][spin-cli] installed.
+
+## Limitations
+
+* The Kubernetes node *os-type* must be Linux.
+* You can't use the Azure portal to deploy SpinKube to an AKS cluster.
+
+## Deploy SpinKube to an existing AKS Cluster
+
+Configure `kubectl` to connect to your Kubernetes cluster using the [az aks get-credentials][az-aks-get-credentials] command. The following command:
+
+```azurecli-interactive
+az aks get-credentials -n myakscluster -g myresourcegroup
+```
+
+### Deploy `cert-manager`
+
+To deploy SpinKube on an AKS cluster, you must have its dependencies installed. SpinKube depends on [cert-manager][cert-manager].
+
+If you haven't deployed `cert-manager` to your AKS cluster yet, you can install it by deploying its Custom Resource Definitions (CRDs), followed by the `cert-manager` Helm chart provided through the `jetstack` repository:
+
+```azurecli-interactive
+# Deploy cert-manager CRDs
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.3/cert-manager.crds.yaml
+
+# Add and update Jetstack repository
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+
+# Install the cert-manager Helm chart
+helm install \
+  cert-manager jetstack/cert-manager --version v1.14.3 \
+  --namespace cert-manager --create-namespace \
+  --wait
+```
+
+### Deploy `runtime-class-manager` (aka KWasm)
+
+The `runtime-class-manager` (aka [KWasm][kwasm]) is responsible for deploying and managing `containerd-shims` on the desired Kubernetes nodes. Deploy it by installing its Helm chart:
+
+```azurecli-interactive
+# Add Helm repository if not already done
+helm repo add kwasm http://kwasm.sh/kwasm-operator/
+# Install KWasm operator
+helm install \
+  kwasm-operator kwasm/kwasm-operator \
+  --namespace kwasm \
+  --create-namespace \
+  --version 0.2.3 \
+  --set kwasmOperator.installerImage=ghcr.io/spinkube/containerd-shim-spin/node-installer:v0.15.1
+```
+
+#### Provision containerd-shim-spin to Kubernetes nodes
+
+Once `runtime-class-manager` is installed on your AKS cluster, you must annotate the Kubernetes nodes that should be able to run Spin Apps with `kwasm.sh/kwasm-node=true`. You can use `kubectl annotate node` to annotate either a subset of the nodes available in your AKS cluster or - as shown in the following snippet - all nodes of your AKS cluster:
+
+```azurecli-interactive
+# Provision containerd-shim-spin to all nodes
+kubectl annotate node --all kwasm.sh/kwasm-node=true
+```
+
+After annotating the Kubernetes node(s), `runtime-class-manager` will use a Kubernetes *Job* to modify the desired node(s). After successful deployment of `containerd-shim-spin`, the node(s) will be labeled with a `kwasm.sh/kwasm-provisioned` label. 
```

Comment:
adding a command to confirm nodes are annoted could help:

`kubectl get nodes --show-labels`

Reply:
Added this @Mossaka